src/components/Header/Header.tsx

```diff
@@ -174,9 +185,11 @@ class Header extends Component<Props, State> {
       case 'help':
         content = (
           // @ts-ignore
-          <IconButton blank={true} color={'inherit'} component={ExternalLink} to={'https://verdaccio.org/docs/en/installation'}>
-            <Help />
-          </IconButton>
+          <ExternalLink blank={true} to="https://verdaccio.org/docs/en/installation">
```

Comment:
Is this correct? We should use ForwardRef to make button behaviour like a anchor link, not wrap the button with a link (accessibility here is a good point)

Reply:
We need a ForwardRef because the component is being used as a child of ToolTip...
The IconButton component hasn't the same props anymore...as before...so I also removed invalid props